types/module/configurator.go

```diff
@@ -47,6 +54,26 @@ type configurator struct {
 	migrations map[string]map[uint64]MigrationHandler
 }
 
+// RegisterService implements Configurator
+func (c configurator) RegisterService(sd *googlegrpc.ServiceDesc, ss interface{}) {
+	fds, err := proto.MergedFileDescriptors()
```

Comment:
```suggestion
	fds, err := proto. MergedRegistry()
```

Also this registry should get cached as a field in `configurator`

Reply:
Thank you!